README.md

```diff
@@ -27,28 +28,30 @@ Orca may undergo changes as we continually enhance and refine the engine to prov
 ## Table of Contents
 
 - [Orca](#orca)
-    - [Table of Contents](#table-of-contents)
-    - [Language Support](#language-support)
-    - [Overview](#overview)
-        - [Custom pronunciations](#custom-pronunciations)
-        - [Voices](#voices)
-        - [Speech control](#speech-control)
-        - [Audio output](#audio-output)
-    - [Demos](#demos)
-        - [Python](#python-demos)
-        - [iOS](#ios-demo)
-        - [C](#c-demos)
-        - [Web](#web-demos)
-        - [Android](#android-demo)
-    - [SDKs](#sdks)
-        - [Python](#python)
-        - [iOS](#ios)
-        - [C](#c)
-        - [Web](#web)
-        - [Android](#android)
-    - [AccessKey](#accesskey)
-    - [Releases](#releases)
-    - [FAQ](#faq)
+  - [Table of Contents](#table-of-contents)
+  - [Overview](#overview)
+    - [Orca streaming text synthesis](#orca-streaming-text-synthesis)
+    - [Text input](#text-input)
+    - [Custom pronunciations](#custom-pronunciations)
+    - [Voices](#voices)
+    - [Speech control](#speech-control)
+    - [Audio output](#audio-output)
+  - [AccessKey](#accesskey)
+  - [Demos](#demos)
+    - [Python Demos](#python-demos)
+    - [iOS Demo](#ios-demo)
+    - [C Demos](#c-demos)
+    - [Web Demos](#web-demos)
+    - [Android Demo](#android-demo)
+  - [SDKs](#sdks)
+    - [Python](#python)
+    - [iOS](#ios)
+    - [C](#c)
+    - [Web](#web)
+    - [Android](#android)
+  - [Releases](#releases)
+    - [v0.1.0 - January 24th, 2024](#v010---january-24th-2024)
```

Comment:
We don't add release details to toc. I know vscode tends to add them automatically; just remove them manually